docs/alexa-apis.rst

```diff
@@ -553,13 +553,14 @@ Use the Alexa Reminders API to create and manage reminders from your skill. This
 Note that you need to modify your skill manifest by adding the reminder permission:
 
 .. code-block:: json
-  ...
-  "permissions": [
-    {
-      "name": "alexa::alerts:reminders:skill:readwrite"
-    }
-  ],
-  ...
+
```

Comment:
this is because there was an error in the code block that caused the sample JSON to not be rendered. https://voxa.readthedocs.io/en/dev/alexa-apis.html#alexa-reminders-api-reference